allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java

```diff
@@ -72,15 +74,43 @@ public void executionStarted(final TestIdentifier testIdentifier) {
 
             methodSource.ifPresent(source -> {
                 result.setDescription(getDescription(source));
-                result.getLabels().add(new Label().withName("suite").withValue(getSuite(source)));
-                result.getLabels().add(new Label().withName("package").withValue(source.getClassName()));
+                result.getLabels().add(new Label().withName(SUITE).withValue(getSuite(source)));
+                result.getLabels().add(new Label().withName(PACKAGE).withValue(source.getClassName()));
             });
 
             getLifecycle().scheduleTestCase(result);
             getLifecycle().startTestCase(uuid);
         }
     }
 
+    @Override
+    public void executionSkipped(TestIdentifier testIdentifier, String reason) {
+        if (testIdentifier.isTest()) {
+            final Optional<MethodSource> methodSource = testIdentifier.getSource()
+                    .filter(MethodSource.class::isInstance)
+                    .map(MethodSource.class::cast);
+            final String uuid = tests.get();
+            final TestResult result = new TestResult()
+                    .withUuid(uuid)
+                    .withName(testIdentifier.getDisplayName())
+                    .withLabels(getTags(testIdentifier))
+                    .withHistoryId(getHistoryId(testIdentifier))
+                    .withStage(Stage.FINISHED)
+                    .withStatus(SKIPPED);
+
+            methodSource.ifPresent(source -> {
```

Comment:
just wondering, why do you need to create a local variable `methodSource`, if it's used only once?.. 89 line could be merged with this one I believe...

Reply:
Don't be so wondering :) such things sometime happens.
Thanks, I'll fix this